yajsapi/executor/index.ts

```diff
@@ -208,40 +205,54 @@ export class Executor {
     payment_driver,
     payment_network,
     event_consumer,
-    network_address
+    network_address,
   }: ExecutorOpts) {
     this._subnet = subnet_tag ? subnet_tag : DEFAULT_SUBNET;
-    this._payment_driver = payment_driver ? payment_driver.toLowerCase() : (driver ? driver.toLowerCase() : DEFAULT_DRIVER);
-    this._payment_network = payment_network ? payment_network.toLowerCase() : (network ? network.toLowerCase() : DEFAULT_NETWORK);
+    this._payment_driver = payment_driver
```

Comment:
hmm... if we're splitting that into multiple lines, wouldn't a regular if/else syntax make more sense?

Reply:
plus, then, the deprecation warning could be included in the if/else and so overall, it would result in even less condition branches...

Reply:
yes, i agree